test/bdd/stress_steps.go

```diff
@@ -381,11 +384,16 @@ func (r *createDIDReq) Invoke() (interface{}, error) {
 		return nil, err
 	}
 
-	logger.Infof("created did successfully %s", intermID)
+	createLogCount++
```

Comment:
I think you need to use atomic.AddInt64, otherwise you'll get a race.

Reply:
done